tools.js

```diff
@@ -59,6 +59,9 @@ module.exports = (function () {
 		timid: {name:"Timid", plus:'spe', minus:'atk'}
 	};
 
+	var init = typeof global.Symbol === 'function' ? Symbol('init') : '_init';
```

Comment:
They're ugly and harder to read and have worse performance and crash if you try to `toString` them and I've never had an actual problem with unintended name collisions.